### ISSUE TITLE
Decoding error - print name of the bad file 

### ISSUE THREAD

**Issue by RDKRACZ** [fhanau/Efficient-Compression-Tool#114]
For some files I'm getting these errors. The problem is I have 16 GB of photos and ECT is not telling me which files it could not decode.
```
Decoding error 28: incorrect PNG signature, it's no PNG or corrupted
Decoding error 57: invalid CRC encountered (checking CRC can be disabled)
```

**Comment by fhanau** [comment]
Providing the file name with the error message is a good idea and will make ECT easier to use when compressing an entire folder. I'll work on implementing this, in the mean time using a script to compress data one file at a time will make it easier to see the affected files.

**Comment by fhanau** [comment]
Implemented in [a29f222](https://github.com/fhanau/Efficient-Compression-Tool/commit/a29f2222754f6e774f533efab9c1cc0b0eb9be10). Thank you for the report!
